core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -406,10 +407,34 @@ public static Duration getDuration(PropertyKey key) {
   }
 
   /**
-   * @return a view of the internal {@link Properties} of as an immutable map
+   * @return a map of the resolved properties(include default properties)
```

Comment:
"a map of the resolved properties represented by this configuration, including all default properties"

Reply:
Done, thanks